frontend/app/src/composables/history/events/notes.ts

```diff
@@ -278,7 +307,7 @@ export function useHistoryEventNote(): UseHistoryEventsNoteReturn {
       formats.push({ type: NoteType.WORD, word: splitted.join('') });
     });
 
-    return formats;
+    return formats.reduce(mergeSequentialWords, new Array<NoteFormat>());
```

Comment:
With the previous approach it seemed like each word was a new text node, with this unless there is something in between the words the should occupy a single one.